src/screens/TokenDetail.js

```diff
@@ -187,10 +220,36 @@ class TokenDetail extends React.Component {
       )
     }
 
+    const renderNftPreview = () => {
+      if (this.state.nft) {
```

Comment:
The whole method is inside this if statement. I feel we should change to:

```javascript
const renderNftPreview = () => {
  if (!this.state.nft) {
    return;
  }
  :
  (here goes the code)
  :
}
```

Reply:
Agreed. But i moved it to a new component.